### DISCUSSION TITLE
Confusion between where to file bugs

### DISCUSSION THREAD

**Discussion by dandv** [PintaProject/Pinta#968] [Q&A] [answered]
I'm *so* glad we can now use GitHub to file bugs! Any chance to disable bug creation [on Launchpad](https://launchpad.net/pinta)?

**Comment by cameronwhite** [comment, marked as answer]
For now I'm keeping Launchpad open until we migrate old bugs that are still valid - it's not clear to me that disabling the bug tracker will keep around the existing issues. Pinta 2.1 also linked to Launchpad from the Help -> File Bug menu so that's another reason to keep it around for a bit

Using Github is the aim going forward though, so please do submit new issues here!
